oshi-core/src/main/java/oshi/hardware/HardwareAbstractionLayer.java

```diff
@@ -122,4 +122,11 @@
      *         present.
      */
     SoundCard[] getSoundCards();
+
+    /**
+     * Returns information about current user
+     *
+     * @return A User object
+     */
+    User getUser();
```

Comment:
I don't think this should be on the HAL, User is a software/OS concept.  Move to the `OperatingSystem`.